src/components/TimeseriesChart.js

```diff
@@ -402,34 +402,83 @@ class TimeseriesChartComponent extends Component {
       annotations = alarmReferenceLines.annotations;
     }
 
-    // Return lines for alarms, ts thresholds and for "now".
-    const nowLine = {
-      type: "line",
-      layer: "above",
-      x0: now,
-      x1: now,
-      yref: "paper",
-      y0: 0,
-      y1: 1,
-      line: {
-        dash: "dot",
-        color: "red",
-        width: isFull ? 2 : 1
+    // Return lines for alarms, ts thresholds and timelines
+    const twoHoursinEpoch = 2 * 60 * 60 * 1000;
```

Comment:
Dit is eigelijk geen epoch formaat, maar gewoon 2 uur uitgedrukt in miliseconds.
dit zou je kunnne noemen const twoHoursMiliseconds